pkgs/tools/package-management/nix/common.nix

```diff
@@ -133,12 +131,12 @@ self = stdenv.mkDerivation {
     nlohmann_json
   ];
 
-  NIX_LDFLAGS = lib.optionals (!atLeast24) [
-    # https://github.com/NixOS/nix/commit/3e85c57a6cbf46d5f0fe8a89b368a43abd26daba
-    (lib.optionalString enableStatic "-lssl -lbrotlicommon -lssh2 -lz -lnghttp2 -lcrypto")
-    # https://github.com/NixOS/nix/commits/74b4737d8f0e1922ef5314a158271acf81cd79f8
-    (lib.optionalString (stdenv.hostPlatform.system == "armv5tel-linux" || stdenv.hostPlatform.system == "armv6l-linux") "-latomic")
-  ];
+#  NIX_LDFLAGS = lib.optionals (!atLeast24) [
```

Comment:
should be removed?

Reply:
I feel like so yes but unsure yet.

Reply:
dropped. I checked pkgsStatic does still build, don't have an armv7 machine to test the build there.